dali/python/backend_impl.cc

```diff
@@ -693,6 +702,10 @@ void ExposeTensorList(py::module &m) {
     .def("layout", [](TensorList<CPUBackend> &t) {
       return t.GetLayout().str();
     })
+    .def("shape", &py_shape_list<CPUBackend>,
```

Comment:
Can you add a test for this in the test_backend_impl.py or similar? Produce a TensorList of known shape and check if it returns it?

Reply:
done, managed to change the type of the return value in the docstring as well.

![Screenshot 2022-01-04 110730](https://user-images.githubusercontent.com/56883920/148043190-bc33da01-7f62-4e15-b80e-05a9ed4e7e74.png)